internal/config/config.go

```diff
@@ -82,6 +83,8 @@ type Pool struct {
 	// unusable, for maximum compatibility with ancient parts of the
 	// internet.
 	AvoidBuggyIPs bool
+	// Prevents IP addresses to be automatically assigned from this pool.
```

Comment:
Can you start the docstring with "if false, ..." here? Otherwise this docstring sounds like I have to set this = true to prevent auto-allocation.